Source/Controllers/SPAppController.m

```diff
@@ -213,7 +213,7 @@ - (void)applicationDidFinishLaunching:(NSNotification *)notification
 
         SPLog(@"We have stale bookmarks");
 
-        NSString *staleBookmarksString = [secureBookmarkManager.staleBookmarks componentsJoinedByString:@"\n"];
+        NSString *staleBookmarksString = [secureBookmarkManager.staleBookmarks componentsJoinedByString:@"\n\n"];
```

Comment:
What if we did something like just display the filenames in the modal popup? Seems a little lengthy with longer paths
```
        NSMutableString *staleBookmarksString = [[NSMutableString alloc] init];
        for (NSString *bookmarkName in secureBookmarkManager.staleBookmarks) {
            [staleBookmarksString appendFormat:@"\n%@", [bookmarkName lastPathComponent]];
        }
```

Reply:
Like this:

<img width="420" alt="Screenshot 2021-01-04 at 5 41 09 PM" src="https://user-images.githubusercontent.com/1576190/103521653-19de2200-4eb4-11eb-837d-f4997d2ee4ae.png">

Reply:
Hmm, not so good on Big Sur:

<img width="372" alt="Screenshot 2021-01-05 at 12 52 46 AM" src="https://user-images.githubusercontent.com/1576190/103560041-14073180-4ef2-11eb-8711-2fa49b3041fd.png">

From [Apple](https://developer.apple.com/design/human-interface-guidelines/macos/overview/whats-new-in-macos/#alerts):

> Alerts are cardlike rectangles that use the same corner radius as all windows in macOS 11. **Within an alert, most content is center aligned.**

Reply:
might need to `[alert setAccessoryView:myViewWithEverything];` on Big Sur....

Reply:
Definitely better with the just-file-name list! But yeah, still a little funky.